### ISSUE TITLE
Root object not exported anymore when using "Selected Objects"

### ISSUE THREAD

**Issue by JiDW** [KhronosGroup/glTF-Blender-IO#1870]
Hello, and thank you for your effort in supporting the Blender GLTF I/O Addon!

**Describe the bug**
When Selecting an object that contains multiple meshes, the resulting export is a scene with all the children at the top level instead of inside their root object. Custom properties attached to the root object are also lost in the process.

This used to work in Blender 3.1, I am unsure of what version of Blender changed this behavior. I did notice the removal of the `export_selected` parameter that I had to use at some point because of issues with the `use_selected` parameters. I do not know if those two things are connected.

**To Reproduce**
1. Create an empty root object
2. Create at least two meshes that have this object as their parent
3. (Opt) Add a custom property to the root object you wish to export
4. Select the root object
5. Export the gltf with "Selected objects" (and "Custom Properties", for the demo purpose)

**Expected behavior**
A gltf file with the root object as the first child  of the scene with its custom properties correctly exported, and the meshes as children of this root object.

**Screenshots**
![blenderexport34](https://user-images.githubusercontent.com/581736/219878314-fceb56fc-0ba1-419d-bc45-2631e77db932.jpg)

Before (3.1)
![blenderexport31gltf](https://user-images.githubusercontent.com/581736/219878443-43b0139c-aac2-4969-b0a3-b885716ef54a.jpg)

Now (3.4)
![blenderexport34gltf](https://user-images.githubusercontent.com/581736/219878450-4373cc81-d7ff-40d9-bb90-3e5e84f83b26.jpg)

**Version**
 - OS: Win 11
 - Blender Version 3.4.1 & 3.5b

**Comment by julienduroure** [comment]
I can't reproduce.

Seems your screenshot is from Blender 2.92
If only the root is selected, only this object is exported, in both 2.92 and 3.4/3.5 (if only selection is activate)
(And of course custom properties of object are export only if this object is selected, and "Custom Properties" option is on)

Please provide your blend file.

**Comment by JiDW** [comment]
Oh no! My bad, I went too fast when trying to reproduce the steps; they do not cause any issues, indeed.

In my actual setup, I use the Python API to export my models. The removal of "export_selected" is indeed the cause of the "change" of behavior I have experienced, but when I tested to reproduce without this parameter, I probably got confused between my test files and mistakenly assumed there was more to it.

I was able to update my script to select the entire hierarchy and remove `export_selected`, and it now works again. Sorry for abusing your time 🙏
